### DISCUSSION TITLE
Sitemap was not being accepted by Google.

### DISCUSSION THREAD

**Discussion by itsidleboy** [MrGreensWorkshop/MrGreen-JekyllTheme#94] [Problems] [answered]
![image](https://user-images.githubusercontent.com/62786811/168519810-2a7857ab-c579-4f7c-9c65-2e07470b44a9.png)

I used the assests/sitemap.xml first but Google shows the error so i used the Jekyll Sitemap plugin and it works better, it will generate sitemap when repo is updated and is accepted by Google within a day.

**Comment by MrGreensWorkshop** [comment, marked as answer]
Without providing sitemap.xml it doesn't prove that it's bug.  There are 2 possibilities. Either something wrong with the site content or bug on site map generation. Why don't you check sitemap xml file with one of online xml validator and find out what's wrong. If it's bug you can create an issue using issue template.

one of online sitemap validator.
https://www.xml-sitemaps.com/validate-xml-sitemap.html

I checked demo site map with it and there is no problem. Also google accepts the sitemap.xml file.
https://mrgreen-jekylltheme.mrgreensworkshop.com/assets/sitemap.xml
